source/blog/2018-07-19-ovn-dpdk.html.md

```diff
@@ -65,14 +65,17 @@ For more information about OVN localnet integration in oVirt, refer to oVirt's [
     # ansible-galaxy install oVirt.dpdk-setup
     ```
 2) Execute the role as described above.
-3) Unless the host was already configured, the host will restart for kernel changes to be applied.
-4) Make sure the following:
+3) Unless the host was already configured, the host will restart for kernel changes to be applied. Note that after reboot the following values are changed (If they are not, you may have incompatible hardware, or you should report a bug on ovirt@users.org):
     * IOMMU: ```/sys/devices/virtual/iommu/<DMAR device>/devices/<PCI address>``` should exist. DMAR devices are typically annotated by dmar0, dmar1, and so forth.
     * hugepages: ```grep Huge /proc/meminfo``` should reflect the desired state as defined in the Ansible playbook (i.e. `Hugepagesize` and `HugePages_Total`)
-    * CPU partitioning: based on the devices that are to be used with a DPDK compatible driver, CPU cores should be partitioned separately on each NUMA node. Refer to `lscpu` to see live CPU NUMA separation information, e.g.: `NUMA node1 CPU(s):     8-15`
-5) Create an oVirt network ("localnet-0"). This network will be used to attach a DPDK NIC.
-6) Create an external OVN network ("external-net-0"), setting "localnet-0" as its physical network. This configuration between "external-net-0" and "localnet-0" will enable traffic between "localnet-0" (which DPDK's physical port is part of) and the rest of the ports present in OVN's "external-net-0" network.
-7) "external-net-0" vNic's may now be added to virtual machines. These vNics now share L2 connectivity with other remote ports in "external-net-0" via DPDK's NIC (and other local ports internally).
-
-Note:
-* Virtual machines using DPDK based NIC's need to have both `hugepages_shared` (to `true`) and `hugepages` (to the amount of requested shared hugepages) custom properties set. Host to VM packets are transmitted and received on buffers allocated on shared hugepages memory. As of date, these properties are not automated.
+    * CPU partitioning: based on the devices that are to be used with a DPDK compatible driver, CPUs should be partitioned separately between each NUMA node. Refer to `lscpu` to see live CPU NUMA separation information, e.g.: `NUMA node1 CPU(s):     8-15`
+4) Create an oVirt network ("phys-net-demo") on your data center and attach it to your cluster.
+![](https://image.ibb.co/mm7LJd/5_create_phys_net.png)
```

Comment:
"Also, don't use Imgbb to host image libraries you link to from elsewhere, content for your website, advertising, avatars, or anything else that turns us into your content delivery network."

please upload your images to ovirt-site (you can include them in your git commit)

Reply:
haha, oops! didn't mean to commit with imgbb

Reply:
Done